automated install/pihole-setupVars.conf.example

```diff
@@ -0,0 +1,35 @@
+# ;
+# ; *  IMPORTANT NOTE:  This file is used as a variable file for the Bash install script and
+# ;    as a PHP ini file for /var/www/html/admin/settings.php.
+# ; *  Bash reads comments as "#"
+# ; *  PHP reads comments as ";" (As of PHP 7.0.0, "#" are no longer recognized as PHP comments)
+# ; *  All lines must end with a LF, not CRLF
+# ; *  There must be a blank line at the end of the file.
+# ; *  Options set during or after installation will be appended to the last line.
+# ;
+# ; *  To create a random Web GUI password, delete "WEBPASSWORD=".  If it is left as is, no password will
+# ;    be required to login.  The password can be changed after installation with 'pihole -a -p'
```

Comment:
Probably it is best to comment `WEBPASSWORD=` by default, to preserve the current default behaviour (random generated password) when this line is not touched?